utils/securityJobSummary.go

```diff
@@ -1,553 +1,600 @@
 package utils
 
 import (
+	"errors"
 	"fmt"
 	"path/filepath"
 	"sort"
 	"strings"
 
-	"github.com/jfrog/gofrog/datastructures"
-	"github.com/jfrog/jfrog-cli-core/v2/commandsummary"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils/commandsummary"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
 	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
 	"github.com/jfrog/jfrog-cli-security/formats"
-	"github.com/jfrog/jfrog-cli-security/formats/sarifutils"
+	"github.com/jfrog/jfrog-cli-security/resources"
 	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
 	"github.com/jfrog/jfrog-cli-security/utils/severityutils"
-	"github.com/jfrog/jfrog-client-go/utils/log"
-	"github.com/jfrog/jfrog-client-go/xray/services"
-	"github.com/owenrumney/go-sarif/v2/sarif"
-	"golang.org/x/exp/maps"
 )
 
 const (
-	Build    SecuritySummarySection = "Builds"
-	Binary   SecuritySummarySection = "Artifacts"
-	Modules  SecuritySummarySection = "Modules"
-	Curation SecuritySummarySection = "Curation"
+	Build    SecuritySummarySection = "Build-info Scans"
+	Binary   SecuritySummarySection = "Artifact Scans"
+	Modules  SecuritySummarySection = "Source Code Scans"
+	Docker   SecuritySummarySection = "Docker Image Scans"
+	Curation SecuritySummarySection = "Curation Audit"
+
+	PreFormat     HtmlTag = "<pre>%s</pre>"
+	ImgTag        HtmlTag = "<img alt=\"%s\" src=%s>"
+	CenterContent HtmlTag = "<div style=\"display: flex; align-items: center; text-align: center\">%s</div>"
+	BoldTxt       HtmlTag = "<b>%s</b>"
+	Link          HtmlTag = "<a href=\"%s\">%s</a>"
```

Comment:
I'd suggest renaming the vars that has more than one %s in them - e.g, SummaryAndDetails.
In Link or ImgTag I'm still not sure myself - but when we have completely two texts - we should be clearer

Reply:
done (you can also hover over it and see the actual string)